Exercise_D and E/assets/style/index.css

```diff
@@ -2,3 +2,54 @@
 This css sheet is linked to practice.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+
+.imgcontainer {
+  display: flex;
+  flex-direction: column;
+  background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)),
+    url(https://i.imgur.com/ibg3zaN.jpg);
+  background-repeat: no-repeat;
+  width: 1280px;
+  height: 720px;
+  color: white;
+  justify-content: center;
+  align-items: center;
+}
+
+button {
+  background-color: #7ed321;
+  color: white;
+  border-radius: 100px !important;
+}
+
+.tablebox {
+  width: 1280px;
+  background-color: #cee7b4;
+  padding-bottom: 100px;
+}
+
+.icon {
+  width: 80px;
+  height: 80px;
+}
+.row {
+  padding: 60px 150px 0px 150px;
+}
+.div {
+  height: 100px;
+  width: 100px;
+}
+
+.icontext {
+  font-size: 28px;
+  margin-left: 30px;
+}
+
+.col-6 {
+  display: flex;
+  align-items: center;
+}
+
+.firstrow {
+  padding-left: 65px;
+}
```

Comment:
The flowershop website doesnt take up the full size of the page as you're hard coding the width with pixels consider using vw or % to take up the full size of the window.